libusb/os/windows_common.c

```diff
@@ -797,12 +797,13 @@ static int windows_handle_transfer_completion(struct usbi_transfer *itransfer)
 	DWORD result, bytes_transferred;
 
 	if (GetOverlappedResult(transfer_priv->handle, &transfer_priv->overlapped, &bytes_transferred, FALSE))
-		result = NO_ERROR;
+		result = transfer_priv->overlapped.Internal;
 	else
 		result = GetLastError();
 
 	usbi_dbg(ctx, "handling transfer %p completion with errcode %lu, length %lu",
 		 USBI_TRANSFER_TO_LIBUSB_TRANSFER(itransfer), ULONG_CAST(result), ULONG_CAST(bytes_transferred));
+	usbi_dbg(ctx, "overlapped Internal is %lu", ULONG_CAST(transfer_priv->overlapped.Internal));
```

Comment:
Looks like some debugging entry (result is logged on previous line)

Reply:
Yes, it can only be different from *result* if GetOverlappedResult() fails, but in that case I guess the value isn't of much interest either. Maybe this usbi_dbg made more sense in an earlier revision, or before the change.

Reply:
> in that case I guess the value isn't of much interest either

definitely not
maybe better remove it

Reply:
Sure. It was there for testing before the actual change was made. For further testing (in particular for the async driver case) it can make sense to log whether GetOverlappedResult() is successful.